Source/WebCore/css/typedom/transform/CSSTranslate.cpp

```diff
@@ -133,6 +134,15 @@ void CSSTranslate::serialize(StringBuilder& builder) const
     builder.append(')');
 }
 
+ExceptionOr<void> CSSTranslate::setZ(Ref<CSSNumericValue> z)
+{
+    if (!z->type().matchesTypeOrPercentage<CSSNumericBaseType::Length>() || z->type().matches<CSSNumericBaseType::Percent>())
```

Comment:
Can't we write !matches< CSSNumericBaseType::Length> instead?

Reply:
Err, yeah, that would a bit nicer, wouldn't it? This is addressed in the revised commit.